chapters/functions.tex

```diff
@@ -295,8 +296,8 @@ \section{Pure Modelica Functions}\label{pure-modelica-functions}
 A function having a formal function parameter that is \lstinline!impure! must be marked \lstinline!pure! or \lstinline!impure!.
 
 \begin{nonnormative}
-Comment: The semantics are undefined if the function call of an
-impure function is part of an algebraic loop.
+Comment: The semantics are undefined if the function call of an impure function is part of an algebraic loop.
+This can occur for the deprecated case of external functions assumed to be pure, or when using \lstinline!pure(expression)! to call an impure function in a pure function.
```

Comment:
```suggestion
Comment: The semantics are undefined if the function call of an impure function is part of an algebraic loop.
This can occur for the deprecated case of external functions assumed to be impure, or when using \lstinline!pure(expression)! to call an impure function in a pure function.
```
Typo!